static/app/views/performance/landing/widgets/components/widgetContainer.tsx

```diff
@@ -97,13 +101,25 @@ const _WidgetContainer = (props: Props) => {
   }, [rest.defaultChartSetting]);
 
   const chartDefinition = WIDGET_DEFINITIONS({organization})[chartSetting];
+
+  // Construct an EventView that matches this widget's definition. The
+  // `eventView` from the props is the _landing page_ EventView, which is different
+  const widgetEventView = props.eventView.clone();
+  widgetEventView.name = chartDefinition.title;
+  widgetEventView.yAxis = chartDefinition.fields[0]; // All current widgets only have one field
+  widgetEventView.display = DisplayModes.PREVIOUS;
+  widgetEventView.fields = ['transaction', 'project', ...chartDefinition.fields].map(
+    fieldName => ({field: fieldName} as Field)
+  );
+
```

Comment:
Might be nice to move this into its own function for clarity. `function getEventViewForActions` ?

Reply:
Done in [28cf24b](https://github.com/getsentry/sentry/pull/36745/commits/28cf24b65d5704c1db8219ed0304f7dd602b03aa) 👌🏻